edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/PolicyUpdate.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// PolicyUpdate is a Data Transfer Object used for sending authorization policy
+    /// definition from EdgeHub core to Mqtt Broker.
+    /// </summary>
+    internal class PolicyUpdate
```

Comment:
Nit: move to PolicyUpdateHandler.cs as a nested class (since this is internal and used only in 1 class)